app/components/moderation-base.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
```

Comment:
Please generate components with `--pod`, so this would live at `app/components/moderation-base/component.js`, and this component's template at `app/components/moderation-base/template.hbs`.